docs/rpc/cardano-rpc-documentation/constructionPayloads.md

```diff
@@ -0,0 +1,150 @@
+    - `metadata` (object, optional): Metadata associated with the sub-network.
+- `operations` (array of objects, required): An array of operation objects, where each object represents a transaction to be included in the Cardano transaction.
+  - `operation_identifier` (object, required): An object containing an index that uniquely identifies the operation.
+    - `index` (number, required): The index of the operation.
+    - `network_index` (number, optional): The network-specific index of the operation.
+  - `related_operations` (array of objects, optional): An array of related operation identifiers if applicable.
+    - `index` (number, optional): The index of the related operation.
+    - `network_index` (number, optional): The network-specific index of the related operation.
+  - `type` (string, required): The type of the operation (e.g., "TRANSFER").
+  - `status` (string, optional): The status of the operation (e.g., "SUCCESS").
+  - `account` (object, required): An object containing information about the account.
+    - `address` (string, required): The Cardano account address associated with the operation.
+    - `sub_account` (object, optional): An optional sub-account object.
+      - `address` (string, optional): The sub-account address.
+      - `metadata` (object, optional): An optional metadata object for the sub-account. If the SubAccount address is not sufficient to uniquely specify a SubAccount, any other identifying information can be stored here. It is important to note that two SubAccounts with identical addresses but differing metadata will not be considered equal by clients.
+    - `metadata` (object, optional): An optional metadata object for the account. Blockchains that utilize a username model (where the address is not a derivative of a cryptographic public key) should specify the public key(s) owned by the address in metadata.
+  - `amount` (object, required): An object containing information about the transaction amount.
+    - `value` (string, required): The value of the transaction amount.
+    - `currency` (object, required): An object specifying the currency details.
+      - `symbol` (string, required): The symbol or code of the currency.
+      - `decimals` (number, required): The number of decimal places for the currency.
+      - `metadata` (object, optional): Any additional information related to the currency itself. For example, it would be useful to populate this object with the contract address of an ERC-20 token.
+    - `metadata` (object, optional): metadata object for the amount.
+  - `coin_change` (object, required): An object containing information about coin changes in the operation.
+    - `coin_identifier` (object, required): An object containing a coin identifier.
+      - `identifier` (string, required): Identifier should be populated with a globally unique identifier of a Coin. In Bitcoin, this identifier would be transaction_hash:index..
```

Comment:
Bitcoin?

Reply:
This is just example and its even mentioned by official Cardano docs